### PR TITLE
Readme Tidyup

### PR DESCRIPTION
**Pull request by fonty1**
docs(readme): API Docs more prominent, fixed broken TOC links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,17 +11,25 @@ By using this client, you can easily interact with the [Streamr](https://streamr
...
 
+[![Build Status](https://travis-ci.com/streamr-dev/streamr-client-javascript.svg?branch=master)](https://travis-ci.com/streamr-dev/streamr-client-javascript)
+
 ----
 
 ## TOC
 
-[Installation](#installation) · [Usage](#usage) · [API Docs](#api-docs) · [Client options](#client-options) · [Authentication options](#authentication-options) · [Message handler callback](#message-handler-callback) · [StreamrClient object](#streamrclient-object) · [Stream object](#stream-object) · [Subscription options](#subscription-options) · [Data Unions](#data-unions) · [Utility functions](#utility-functions) · [Events](#binding-to-events) · [Partitioning](#partitioning) · [Logging](#logging) · [NPM Publishing](#publishing-latest)
+[API Docs](#API-docs) · [Installation](#installation) · [Usage](#usage) · [Client options](#client-options) · [Authentication](#authentication) · [Managing subscriptions](#managing-subscriptions) · [Stream API](#stream-api) · [Subscription options](#subscription-options) · [Data Unions](#data-unions) · [Utility functions](#utility-functions) · [Events](#events) · [Stream Partitioning](#streamr-partitioning) · [Logging](#logging) · [NPM Publishing](#publishing-latest)
```
Comment (fonty1) [inline README.md:32]:
```suggestion
[API Docs](#API-docs) · [Installation](#installation) · [Usage](#usage) · [Client options](#client-options) · [Authentication](#authentication) · [Managing subscriptions](#managing-subscriptions) · [Stream API](#stream-api) · [Subscription options](#subscription-options) · [Data Unions](#data-unions) · [Utility functions](#utility-functions) · [Events](#events) · [Stream Partitioning](#stream-partitioning) · [Logging](#logging) · [NPM Publishing](#publishing-latest)
```

**Review by timoxley — APPROVED** [review]
Links definitely better, not sure API docs *section* needs to be right at the top though.

**Comment by timoxley** [comment]
> docs(readme): API Docs more prominent, fixed broken TOC links

@fonty1 are you trying to trick me into not seeing the commit message formatting 😃 ??

File: README.md [hunk 2]
```diff
@@ -347,7 +341,7 @@ All the below functions return a Promise which gets resolved with the result.
 
 ## Data Unions
 
-This library provides functions for working with Data Unions.
+This library provides functions for working with Data Unions. Please see the [API Docs](https://streamr-dev.github.io/streamr-client-javascript/) for auto-generated documentation on each Data Union endpoint.
```
[the remark this answers is not in the document]
Comment (fonty1) [reply README.md:344]: I'm a bit worried about those URLs shifting without notice, either by TypeDoc updates or DU class reorg

**Comment by fonty1** [comment]
> > docs(readme): API Docs more prominent, fixed broken TOC links
> 
> @fonty1 are you trying to trick me into not seeing the commit message formatting 😃 ??

@timoxley that got auto inserted into the PR description. I did the commits properly too though right?! 🙈 I do readme changes on the GitHub site because of their previewer + convenience.

File: README.md [hunk 3]
```diff
@@ -11,17 +11,25 @@ By using this client, you can easily interact with the [Streamr](https://streamr
 
 The client uses websockets for producing and consuming messages to/from streams. It should work in all modern browsers.
 
-[![Build Status](https://travis-ci.com/streamr-dev/streamr-client-javascript.svg?branch=master)](https://travis-ci.com/streamr-dev/streamr-client-javascript)
+### API Docs
+
+The [API docs](https://streamr-dev.github.io/streamr-client-javascript/) are automatically generated from the TypeScript source code. They can also be rebuilt locally via:
```
[the remark this answers is not in the document]
Comment (fonty1) [reply README.md:16]: I confess to being extraordinarily ambivalent to where it rests :)  Before Client Options + quick link up top works for me.

**Comment by timoxley** [comment]
>  I did the commits properly too though right?!

image

Only that last one. But it's ok! 👍

**Pull request merged**